qa/suites/krbd/rbd/conf.yaml

```diff
@@ -4,4 +4,4 @@ overrides:
       global:
         ms die on skipped message: false
       client:
-        rbd default features: 37
+              rbd default features: 1
```

Comment:
@dillaman on rhel 8 it seems we can only use layering, is this ok for you?

Reply:
(1) you cannot inject random whitespace into YAML. (2) support for this is forthcoming eminently so we don’t want to just globally disable these tests just because it make life harder for you — the truth is that upstream krbd tests only validate the latest kernels and not random distro kernels.

1.

Reply:
@dillaman FYI this is not for master branch, this is internal test branch to run test again nautilus downstream builds. I understand upstream runs with latest kernels but my point here is we cannot run this for supported downstream builds until we make this change and the tests are *red*, so hence asking for input if there is an alternative or we will go ahead with this change for downstream builds.

Reply:
Why is RH tracking its own QA branch in the upstream repo? This doesn’t make any sense to me. Also, see point (1) where this was just a bad change and for (2) we can talk in a non-public forum about RH release schedules to support these features.

Reply:
This was discussed with @jdurgin and @neha-ojha, The original branch was internal and in gitlab. It gives better visibility to what is running in downstream QE and the challenges seen by downstream (which could be similar to other downstream runs in different company). 

Personally I am not discussing any release schedules, we want to align our scripts and framework so that minimal changes are required for us to run in downstream and major contributions happens to upstream from everyone.

I will setup a meeting so that everyone is on same page, Sorry for any confusion.